### PR TITLE
Add docs for accessing thanos metrics

### PR DESCRIPTION
**Pull request by 4n4nd**
Inspired from [Internal Data Hub docs](https://gitlab.cee.redhat.com/data-hub/dh-docs/-/blob/master/docs/interacting-with-telemetry-data.adoc)
Resolves https://github.com/operate-first/apps/issues/1361

### CODE REVIEW

File: docs/content/observatorium/thanos/README.md [hunk 1]
```diff
@@ -1,15 +1,28 @@
-# Thanos Access
+# Interacting with Operate First Thanos data
```
Comment (HumairAK) [inline docs/content/observatorium/thanos/README.md:1]: Can we keep the title short, as this is converted to a navbar title: 

image

File: docs/content/observatorium/thanos/thanos_programmatic_access.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The Thanos metrics can be queried using the [this](https://prometheus-api-client-python.readthedocs.io/) Python client library.
+You can install the latest version with `pip install prometheus-api-client`.
+You will need to extract the OpenShift bearer token for the client library to authenticate the [thanos](https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/) instance.
+
+### Personal token
+
+You can use your personal token for testing and ad-hoc scripts, but it will expire in ~24 hours.
+
+1. Login to https://console-openshift-console.apps.smaug.na.operate-first.cloud/ with your credentials
```
Comment (suppathak) [inline docs/content/observatorium/thanos/thanos_programmatic_access.md:11]: @4n4nd , Is it possible to add a statement for **Log in with** `kube::admin` or `operate-first`?

File: docs/content/observatorium/thanos/README.md (hunk 1 above)
Comment (4n4nd) [reply docs/content/observatorium/thanos/README.md]: updated :+1:

File: docs/content/observatorium/thanos/thanos_programmatic_access.md (hunk 2 above)
Comment (4n4nd) [reply docs/content/observatorium/thanos/thanos_programmatic_access.md]: updated :+1:

**Comment by suppathak** [comment]
lgtm!

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**